tutorials/napps/restful_statistics.rst

```diff
@@ -139,7 +139,7 @@ Finally, let's generate 5 packets per second from host h1 to h2 with:
 
 .. code-block:: console
 
-  mininet> h1 ping -i 0.2 h2
+  mininet> h1 ping -i 0,2 h2
```

Comment:
This is a iptools locale problem not related to kytos, so it should be removed from the PR.

Reply:
OR, you can add a note about how to run it if the user has problems with this notation